src/textField/story.tsx

```diff
@@ -4,24 +4,23 @@ import { storiesOf } from '@storybook/react'
 import { action } from '@storybook/addon-actions'
 import { withKnobs, text, boolean, number, select } from '@storybook/addon-knobs'
 
-import TextField from 'textField'
+import TextField, { inputTypes, inputModes } from 'textField'
```

Comment:
You can also use them as class constants (ex: `TextField. INPUT_TYPES`) and not export the enums.